js/main.js

```diff
@@ -0,0 +1,158 @@
+let nextEntryId = parseInt(localStorage.getItem('nextEntryId') || 1);
+
+// create the placeholder imageURL so that it can reset back to this
+const placeholderImageURL = '/images/placeholder-image-square.jpg';
+
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I've noticed you're using the event listener for `DOMContentLoaded` - I don't think it's necessary for this project, you can just put all the logic in it in the global scope of `main.js`